metricbeat/module/system/memory/_meta/fields.yml

```diff
@@ -13,59 +13,18 @@
       description: >
         Used memory.
 
-    - name: free
-      type: long
-      description: >
-        Available memory.
-
     - name: used.pct
       type: half_float
       description: >
         The percentage of used memory.
 
-    - name: actual
-      type: group
+    - name: free
+      type: long
       description: >
-        Actual memory fields.
-      fields:
-        - name: used.bytes
-          type: long
-          description: >
-            Actual used memory. This value is the "used" memory minus the memory used for disk caches and buffers.
-            Available only on Unix.
-
-        - name: free
-          type: long
-          description: >
-            Actual available memory. This value is the "free" memory plus the memory used for disk caches and
-            buffers. Available only on Unix.
+        Free memory. For a human readable value use Available.
 
-        - name: used.pct
-          type: half_float
-          description: >
-            The percentage of actual used memory.
-
-    - name: swap
-      type: group
-      prefix: "[float]"
-      description: This group contains statistics related to the swap memory usage on the system.
-      fields:
-        - name: total
-          type: long
-          description: >
-            Total swap memory.
-
-        - name: used.bytes
-          type: long
-          description: >
-            Used swap memory.
-
-        - name: free
-          type: long
-          description: >
-            Available swap memory.
-
-        - name: used.pct
-          type: half_float
-          description: >
-            The percentage of used swap memory.
+    - name: available
+      type: long
+      description: >
+        Available memory is what it matters. It is calculated based on the OS. On Linux it is a sum of free memory
```

Comment:
I think describing why it matters would be better for the description. Also I think we should mention the units are in bytes (we don't currently do this consistently).